_data/people/juliaboltz.yml

```diff
@@ -0,0 +1,3 @@
+username: juliaboltz
+name: juliaboltz
+movie: I don't have a favorite movie
```

Comment:
Do you have a favorite YouTube video or perhaps a favorite music video you'd like to recommend instead?